python/pytests/pandas_source_test.py

```diff
@@ -1,37 +1,41 @@
-import random
-
 import kaskada as kd
 import pandas as pd
 
+def test_float_microseconds(golden) -> None:
```

Comment:
New tests taking floating point numbers in different precisions. Only this test changed -- and only this tests golden files are substantively different.

Reply:
the actual time from this should be something like `Mon Dec 19 2022 19:17:52 GMT+0000`